mobify/astro-scaffold
# Support pending navitron 1.0 release

**NOTE: This PR is not yet complete. It is awaiting the final release of Navitron to npm**

Updates the scaffold to work with the [navitron 1.0](https://github.com/mobify/navitron/pull/7) release

JIRA: N/A
Linked PRs: https://github.com/mobify/navitron/pull/7

## Changes
- Change where we pull the navitron files from (node_modules instead of bower_components)
- Remove dead `run script` in package.json

## How to test-drive this PR
- Run `npm install`
- Open the app in Xcode and run
- Open side menu and ensure the side menu works.

## TODOS:
- [s] CHANGELOG.md has been updated.
